### ISSUE TITLE
aXe: Checkbox Nested in Option is being flagged for nested interactive

### ISSUE THREAD

**Issue by jake-costa** [jpmorganchase/salt-ds#4737]
### Latest version

- [x] I have tested the latest version

### Description

Options that include nested checkbox inputs (e.g. multi-select) are being flagged for [nested-interactive](https://dequeuniversity.com/rules/axe/4.10/nested-interactive?application=AxeChrome)

Components effected:
Multi-select combo box
Multi-select dropdown
Multi-select listbox

In addition, Nvda/Firefox is failing to convey which options are selected/not-selected.

### Steps to reproduce

Open chrome
Navigate to one of the components listed above
ensure the listbox with options containing the checkboxes is displayed
run aXe

### Package name(s)

Core (@salt-ds/core)

### Browser

- [x] Chrome
- [x] Safari
- [x] Firefox
- [ ] Microsoft Edge

### Operating system

- [x] macOS
- [x] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Are you a JPMorgan Chase & Co. employee?

- [x] I am an employee of JPMorgan Chase & Co.
